tools/oeedger8r/CMakeLists.txt

```diff
@@ -37,13 +36,23 @@ add_custom_command(
     DEPENDS ${OEEDGER8R_SRC}
 )
 
+if (WIN32)
```

Comment:
Since this part is exactly the same as the UNIX path aside from the command, does the old way of setting `BUILD_COMMAND` not work? Like `set(BUILD_COMMAND powershell -Command ...  ... ...)`?

Reply:
setting build command that way seemed to create problems with quote escaping.  The command appeared in the log, but did not execute.  Using command directly worked.